fsqual.cc

```diff
@@ -57,7 +57,11 @@ void test_concurrent_append(io_context_t ioctx, int fd, unsigned iodepth, std::s
     while (completed < nr) {
         auto i = unsigned(0);
         while (initiated < nr && current_depth < iodepth) {
-            io_prep_pwrite(&iocbs[i++], fd, buf, bufsize, bufsize*initiated++);
+            io_prep_pwrite(&iocbs[i], fd, buf, bufsize, bufsize*initiated++);
+            if (nowait) {
+                iocbs[i].aio_rw_flags |= RWF_NOWAIT;
```

Comment:
Need to protect against RWF_NOWAIT not being defined on old systems.

Reply:
Shouldn't we handle failures and resubmit them? Or at least count and print them?